src/WalkingModule/src/Module.cpp

```diff
@@ -124,6 +146,48 @@ bool WalkingModule::setRobotModel(const yarp::os::Searchable& rf)
 
 bool WalkingModule::configure(yarp::os::ResourceFinder& rf)
 {
+
+        m_pushRecoveryActiveIndex=0;
+        m_useStepAdaptation = rf.check("use_step_adaptation", yarp::os::Value(false)).asBool();
+        // TODO REMOVE ME
+        impactTimeNominal = 0;
+        impactTimeAdjusted = 0;
+
+        zmpNominal.zero();
+        zmpAdjusted.zero();
+        m_isPushActive=0;
+
+        iDynTree::Position tempTemp;
+        iDynTree::Rotation tempRot;
+        tempRot.Identity();
+        //    tempRot.
+        tempTemp.zero();
+        m_adaptatedFootLeftTransform.setPosition(tempTemp);
+        m_currentFootLeftTransform.setPosition(tempTemp);
+        m_smoothedFootLeftTransform.setPosition(tempTemp);
+
+        m_adaptatedFootRightTransform.setPosition(tempTemp);
+        m_currentFootRightTransform.setPosition(tempTemp);
+        m_smoothedFootRightTransform.setPosition(tempTemp);
+
+        m_currentFootLeftTransform.setRotation(tempRot);
+        m_adaptatedFootLeftTransform.setRotation(tempRot);
+        m_smoothedFootLeftTransform.setRotation(tempRot);
+        m_currentFootRightTransform.setRotation(tempRot);
+        m_adaptatedFootRightTransform.setRotation(tempRot);
+        m_smoothedFootRightTransform.setRotation(tempRot);
```

Comment:
Can you use some more descriptive names? Like ``zeroPosition`` and ``identityRotation``.

Also, notice that you can do
```
m_adaptedFootLeftTransform = iDynTree::Transform::Identity();
```

This would apply also to the other transforms.

Reply:
Done!